recipes/R/Retail - GradientBoosting/retail.config.json

```diff
@@ -68,7 +68,7 @@
         "name": "score",
         "parameters": [
             {
-                "key": "output_dataset_id",
+                "key": "scoringResultsDataSetId",
```

Comment:
This config can be removed entirely. I'm also referring to your question in the Jira ticket. `scoringResultsDataSetId ` is not picked by the UI, by it is set by the UI. When you choose a scoring results dataset, the UI passes in the id for that dataset as `scoringResultsDataSetId` in the configs. And as Matt mentioned in the Jira, which scoring results datasets you see in the UI depend on `ACP_DSW_SCORING_RESULTS_XDM_SCHEMA`. But due to some recent UI improvements, this config is also set by the UI when you are creating a recipe and choose an output schema, so you should actually be able to remove that config too.

Reply:
Thank you Shiv for explaining this.